### PR TITLE
Support for dark mode battery icon

### PR DESCRIPTION
**Pull request by zats**
Hi, I don't have a lot of experience with mac development, so please do be critical when reviewing this code 🙂

This PR results in following:
screen shot 2016-05-26 at 9 09 21 am screen shot 2016-05-26 at 9 09 27 am

### CODE REVIEW

File: MenubarlessClock/MBLCAppDelegate.m [hunk 1]
```diff
@@ -373,6 +384,7 @@ - (void)	adaptUIToDarkMode
 		self.window.backgroundColor = [NSColor colorWithWhite: 1.0 alpha: 0.9];
 		self.timeField.textColor = [NSColor blackColor];
 	}
+    [self updateClock:nil];
```
Comment (zats) [inline MenubarlessClock/MBLCAppDelegate.m]: Here I intended to trigger battery icon invalidation but couldn't find a better way. 
Please let me know if you can think of a better one!

**Comment by dbergey** [comment]
@uliwitness any chance you could merge this one? battery icon is invisible in dark mode current build. 😄

**Comment by uliwitness** [comment]
Hi, this is a useful patch, but sadly it looks like it obliterates the red fill when battery is low. Any chance of making it not turn that red part white? Also, I tried just using MacOS' "template" flag on for the images, but can't make it work.

**Comment by zats** [comment]
Good point! I fixed that
screen shot 2016-06-25 at 10 41 23 am
screen shot 2016-06-25 at 10 41 13 am
screen shot 2016-06-25 at 10 41 04 am
screen shot 2016-06-25 at 10 40 42 am

And yes, I tried template images at first too, didn't quite work for me either

**Comment by dbergey** [comment]
@uliwitness any chance of this ever being merged, or does it need a re-work?

**Review by uliwitness — APPROVED** [review]

**Comment by zats** [comment]
👍🔥

**Pull request merged**